### PR TITLE
Fix node operator doc quote

### PR DESCRIPTION
**Pull request by ebmifa**
## Description 
Fix node operator doc quote

## Test plan 
Before:
Screenshot 2024-10-15 at 3 31 55 PM

After:
Screenshot 2024-10-15 at 3 32 00 PM

### CODE REVIEW

**Review by johnjmartin — APPROVED** [review]

**Review by ronny-mysten — CHANGES_REQUESTED** [review]
We don't use quote blocks. Why do you hate the info box so much?

**Comment by ebmifa** [comment]
> We don't use quote blocks. Why do you hate the info box so much?

@ronny-mysten did you see how they got rendered? Look at my test plan

File: docs/content/guides/operator/updates.mdx [hunk 1]
```diff
@@ -11,9 +11,7 @@ Each Sui network is deployed on a consistent schedule. There are extenuating cir
 - `testnet`: Deployed every week on Tuesdays.
 - `mainnet`: Deployed every two weeks on Wednesdays.
 
-:::info
- For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
- :::
+> For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:14]:
```suggestion
:::info
For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
:::
```
Oh, I think there was an added space messing it up?

**Review by ronny-mysten — APPROVED** [review]
You'll learn to love the info box :) Thanks @ebmifa !

**Pull request merged**